src/main/java/org/apache/skywalking/banyandb/v1/client/MeasureQuery.java

```
@@ -107,27 +122,50 @@ BanyandbMeasure.QueryRequest build() throws BanyanDBException {
         builder.setMetadata(buildMetadata());
         if (timestampRange != null) {
             builder.setTimeRange(timestampRange.build());
+        } else {
+            builder.setTimeRange(TimestampRange.MAX_RANGE);
```

@hanahmily Should we consider this as a temp fix?

---

Or, should we report an error in measure?

---

I'm amending the server side with a default time range. Could we hold this PR for a while?

---

Sure, we could wait for your side work.